### DISCUSSION TITLE
【preset-umi】这个包，exportsFields 为什么要特意写成 [] 呢？

### DISCUSSION THREAD

**Discussion by wzy2023** [umijs/umi#11593] [Q&A]
<img width="1176" alt="image" src="https://github.com/umijs/umi/assets/116612814/e514fd8a-a5aa-4a17-9762-252b56869ce2">

可否去掉呢，这样才能支持 package.json 里用 exports 导出的包

**Comment by fz6m** [comment]
正常的包都要包含 `main` 字段，如有问题，需要给一个最小复现。
